airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -38,19 +38,6 @@ data:
         message: "Add default cursor for cdc"
         upgradeDeadline: "2023-08-23"
   connectorTestSuitesOptions:
-    - suite: unitTests
```

Comment:
I understand the removal of `integrationTests` but is removing `unitTests` intentional?